packages/lxc.rb

```diff
@@ -0,0 +1,57 @@
+require 'buildsystems/meson'
+
+class Lxc < Meson
+  description 'Linux Containers'
+  homepage 'https://linuxcontainers.org'
+  version '6.0.0'
+  license 'LGPL'
+  compatibility 'all'
+  source_url 'https://github.com/lxc/lxc.git'
+  git_hashtag "v#{version}"
+
+  depends_on 'libseccomp' # R
+  depends_on 'libcap'     # R
+  depends_on 'wget2'      # needed by lxc-download
+
+  meson_options <<~EOF
+    -Dlocalstatedir=#{CREW_PREFIX}/var \
+    -Druntime-path=#{CREW_PREFIX}/run \
+    -Dinit-script=[] \
+    -Ddbus=false \
+    -Dapparmor=false \
+    -Dselinux=false \
+    -Dexamples=false \
+    -Dtests=false \
+    -Dman=false \\
+  EOF
+
+  lxc_start_wrapper = <<~EOF
+    #!/bin/bash -eu
+    mkdir -p #{CREW_PREFIX}/run /sys/fs/cgroup/systemd
+
+    if ! mountpoint -q /sys/fs/cgroup/systemd; then
+      # systemd-based distros will not boot without this
+      mount cgroup -t cgroup -o none,name=systemd /sys/fs/cgroup/systemd
+    fi
+
+    exec lxc-start.real "${@}"
+  EOF
+
```

Comment:
Should we have a preflight check in order to ensure that the kernel has been recompiled as required?